src/client/package.json

```diff
@@ -158,7 +158,9 @@
     "webpack": "4.39.1"
   },
   "optionalDependencies": {
-    "openfin-cli": "^2.0.4"
+    "glue4office": "1.4.5",
```

Comment:
if possible, can these be `npm` packages? as opposed to having to manually symlink them?

Reply:
Yes, the glue.js package is published as `@glue42/desktop` and the Glue4Office package is planned to be published soon:

```
  "@glue42/desktop": "4.5.7",
  "glue4office": "file:./glue4office-1.4.8.tgz",
```